src/components/Operatives/OperativeWorkOrderDetails.js

```diff
@@ -146,6 +198,7 @@ OperativeWorkOrderDetails.propTypes = {
   property: PropTypes.object.isRequired,
   workOrder: PropTypes.instanceOf(WorkOrder).isRequired,
   personAlerts: PropTypes.array.isRequired,
+  locationAlerts: PropTypes.array.isRequired,
 }
 
 export default OperativeWorkOrderDetails
```

Comment:
Since we are using in both places `getCautionaryAlertsType(locationAlerts, personAlerts)`, we can create a variable and store it. Something like that:
```js
const cautionaryAlertsType = getCautionaryAlertsType(locationAlerts, personAlerts)
```

By doing that on Line 121, it can be
```js
cautionaryAlertsType && ( etc )
```

Reply:
what this storing will give us?

Reply:
ok, I see

Reply:
We won’t call twice the `getCautionaryAlertsType(locationAlerts, personAlerts)`